Source/WebCore/rendering/style/StyleContentAlignmentData.cpp

```diff
@@ -35,4 +35,57 @@ TextStream& operator<<(TextStream& ts, const StyleContentAlignmentData& o)
     return ts << o.position() << " " << o.distribution() << " " << o.overflow();
 }
 
+bool StyleContentAlignmentData::isStartward(bool isRTL, bool isFlexReverse) const
```

Comment:
Instead of boolean arguments, this should use the relevant enum classes (TextDecoration instead of bool isRTL, whatever the enum for flexdirection is for the other one). boolean arguments are ambiguous at their callsites.

Reply:
I think that's a good call for TextDecoration. For flexDirection, I'm going to leave it as a boolean because it has to work in both axes and the flex properties that control this are not so simple. It's better for the Flexbox caller to handle that logic and pass it through here; I want to avoid putting too much flex logic in this API.